spec/lib/tasks/change_history_spec.rb

```diff
@@ -0,0 +1,112 @@
+RSpec.describe "Change History tasks" do
+  include ActiveJob::TestHelper
+
+  before do
+    populate_default_government_bulk_data
+  end
+
+  describe "change_history:show" do
+    let(:change_history_entry) do
+      {
+        "id": SecureRandom.uuid,
+        "public_timestamp": "2020-02-20T16:00:00+00:00",
+        "note": "Test",
+      }
+    end
+    let(:first_edition) do
+      create(:edition,
+             current: false,
+             first_published_at: "2020-02-20 12:00:00",
+             locale: "en")
+    end
+    let(:second_edition) do
+      create(:edition,
+             document: first_edition.document,
+             number: 2,
+             locale: "en",
+             change_history: [change_history_entry])
+    end
+
+    before do
+      allow($stdout).to receive(:puts)
+      Rake::Task["change_history:show"].reenable
+    end
+
+    it "prints change history without a locale" do
+      expect {
+        Rake::Task["change_history:show"].invoke(second_edition.document.content_id)
+      }.to output(
+        "#{change_history_entry[:id]} | 2020-02-20T16:00:00+00:00 | Test\n",
+      ).to_stdout
+    end
+
+    it "prints change history with a locale" do
+      first_edition = create(:edition,
+                             current: false,
+                             first_published_at: "2020-02-20 12:00:00",
+                             locale: "cy")
+      second_edition = create(:edition,
+                              document: first_edition.document,
+                              number: 2,
+                              locale: "cy",
+                              change_history: [change_history_entry])
+
+      expect {
+        Rake::Task["change_history:show"].invoke(second_edition.document.content_id, "cy")
+      }.to output(
+        "#{change_history_entry[:id]} | 2020-02-20T16:00:00+00:00 | Test\n",
+      ).to_stdout
+    end
+  end
+
+  describe "change_history:delete" do
+    let(:change_history_modifier) { instance_double(ChangeHistoryModifier) }
+
+    before do
+      Rake::Task["change_history:delete"].reenable
+      allow(ChangeHistoryModifier).to receive(:new).and_return(change_history_modifier)
+      allow(change_history_modifier).to receive(:delete)
```

Comment:
I don't think you need this line as it's covered by the expect on line 72, same for other tests.

Reply:
Will fix.